coalib/parsing/CliParsing.py

```diff
@@ -65,16 +65,19 @@ def parse_cli(arg_list=None,
                                   origin,
                                   line_parser)
         else:
+            if arg_key == 'bears':
+                if isinstance(arg_value, list):
+                    arg_value = [
```

Comment:
You can instead do ``if arg_key == 'bears' and isinstance(arg_value, list):``. 
Nested if is not required here.